config/routes.rb

```diff
@@ -1,13 +1,6 @@
 Rails.application.routes.draw do
   root to: redirect("/content")
 
-  get "/healthcheck", to: proc { [200, {}, %w[OK]] }
-  get "/healthcheck-f",
-      to: proc {
-            GovukError.notify("Sentry works")
-            [200, {}, ["Sentry notified"]]
-          }
```

Comment:
Do we have/need an alternative to this?

Reply:
As far as I can tell it's not checked by anything